src/psyclone/tests/gocean1p0_opencl_test.py

```diff
@@ -161,6 +176,7 @@ def test_psy_init(kernel_outputdir):
 
 
 def test_opencl_options_validation(kernel_outputdir):
+    # pylint: disable=unused-argument
```

Comment:
You can do `@pytest.mark.usefixtures("kernel_outputdir")` I've learned (and then we don't have to specify `kernel_outputdir` as an argument to the test). Please do this instead of the pylint-disable directives.

Reply:
Oh, learned another thing. Not as great as 'named tuple', but still not bad :)